custom_components/tuya_local/devices/moes_wht009_thermostat.yaml

```diff
@@ -0,0 +1,226 @@
+name: Moes thermostat WHT-009
+products:
+  - id: 14bmxdarlb4ravgd
+    name: Moes thermostat WHT-009
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          value: heat
+    - id: 2
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: manual
+          value: Manual
+        - dps_val: auto
+          value: Auto
+    - id: 16
+      name: temperature
+      type: integer
+      unit: C
+      range:
+        min: 50
+        max: 450
+      mapping:
+        - scale: 10
+          step: 5
+    - id: 24
+      name: current_temperature
+      type: integer
+      mapping:
+        - scale: 10
+          step: 5
+    - id: 19
+      type: integer
+      name: max_temperature
+      mapping:
+        - scale: 10
+    - id: 26
+      type: integer
+      name: min_temperature
+      mapping:
+        - scale: 10
+    - id: 36
+      type: string
+      name: hvac_action
+      mapping:
+        - dps_val: open
+          value: heating
+        - dps_val: close
+          value: idle
+    - id: 71
+      type: string
+      optional: true
+      name: week_program
+
+secondary_entities:
+  - entity: number
+    name: Maximum temperature
+    class: temperature
+    category: config
+    icon: "mdi:thermometer-chevron-up"
+    dps:
+      - id: 19
+        type: integer
+        name: value
+        optional: true
+        unit: C
+        range:
+          min: 150
+          max: 450
+        mapping:
+          - scale: 10
+            step: 10
+  - entity: sensor
```

Comment:
There is no need to repeat the current_temperature and hvac_action as separate sensors.